qemu/tests/cfg/qemu_guest_agent_hotplug.cfg

```diff
@@ -15,12 +15,12 @@
     gagent_check_type = sync
     # /qga_setup_fail.log is saved during guest preinstall phase, if you don't have this file, please skip this parameter.
     cmd_check_qga_installlog = "ls /qga_setup_fail.log"
-    extra_params = "-device virtio-serial-pci,id=qga-virtio-serial0,bus=pci.0"
+    extra_params = "-device '{"driver":"virtio-serial-pci","id": "qga-virtio-serial0","bus":"pci.0"'"
     q35:
         # temporary workaround since currently no virtio-serial-pci bus object provided in the framework
         pci_controllers += " pcie_root_port_qga"
         type_pcie_root_port_qga = "pcie-root-port"
-        extra_params = "-device virtio-serial-pci,id=qga-virtio-serial0,bus=pcie_root_port_qga"
+        extra_params = "-device '{"id": "qga-virtio-serial0", "driver": "virtio-serial-pci", "bus": "pcie_root_port_qga"}'"
```

Comment:
```suggestion
        serial_type = "virtserialport"
```

Reply:
thanks , it works for x86_64: 
 (1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.qemu_guest_agent_hotplug.q35: PASS (45.08 s)
 Could you guys please help to run it? @fbq815 @BohdanMar ， to check whether it can be passed for your architecture

Reply:
Tescase Pass after adding aarch64 suporting to the platfrom list in .cfg JOB ID     : 1874d692326e60aa391b9fdd67d9ef123905e168
JOB LOG    : /root/avocado/job-results/job-2023-03-15T09.17-1874d69/job.log
 (1/2) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.arm64-pci: STARTED
 (1/2) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.arm64-pci: PASS (627.41 s)
 (2/2) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.qemu_guest_agent_hotplug.arm64-pci: STARTED
 (2/2) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.qemu_guest_agent_hotplug.arm64-pci: PASS (63.88 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2023-03-15T09.17-1874d69/results.html
JOB TIME   : 693.22 s